doc/user/BuildConfiguration.md

```diff
@@ -952,6 +961,116 @@ The named set of mapping rules applied when parsing OSM tags. Overrides the valu
 
 The named set of mapping rules applied when parsing OSM tags.
 
+<h3 id="transferParametersForMode">transferParametersForMode</h3>
+
+**Since version:** `2.7` ∙ **Type:** `enum map of object` ∙ **Cardinality:** `Optional`   
+**Path:** /   
+**Enum keys:** `not-set` | `walk` | `bike` | `bike-to-park` | `bike-rental` | `scooter-rental` | `car` | `car-to-park` | `car-pickup` | `car-rental` | `car-hailing` | `flexible`
+
+Configures mode-specific properties for transfer calculations.
+
+This field enables configuring mode-specific parameters for transfer calculations.
+To configure mode-specific parameters, the modes should also be used in the `transferRequests` field in the build config.
+
+**Example**
+
+```JSON
+// build-config.json
+{
+  "transferParametersForMode": {
+    "CAR": {
+      "disableDefaultTransfers": true,
+      "carsAllowedStopMaxTransferDuration": "3h"
+    },
+    "BIKE": {
+      "maxTransferDuration": "30m",
+      "carsAllowedStopMaxTransferDuration": "3h"
+    }
+  }
+}
+```
+
+
+<h3 id="tpfm_BIKE_carsAllowedStopMaxTransferDuration">carsAllowedStopMaxTransferDuration</h3>
```

Comment:
Now the same fields are documented for all modes (specified in the example build-config.json) which is not what we want since I doubt we will have mode specific parameters. I don't know what is the best way to get rid of this. One option would be to have a structure like:
```
{
  "transferParametersForModes": [
    {"mode": "CAR", "maxTransferDuration": "30m"}
  ]
}
```
instead of having this kind of an enum map.

Reply:
As discussed this has been left as is. I added a topic discussion in the dev meeting thread